projects/1.20/assets/Warriors of Past Epoch/warriorsofpastepoch/lang/zh_cn.json

```
@@ -0,0 +1,54 @@
+{
+  "effect.warriorsofpastepoch.disease_resistance": "疫疾抗性",
+  "effect.warriorsofpastepoch.contagious_touch_morbidity": "感染之触:染疫",
```

标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。


```suggestion
  "effect.warriorsofpastepoch.contagious_touch_morbidity": "感染之触：染疫",
```